pkg/model/model.go

```diff
@@ -38,6 +40,103 @@ type Coeff struct {
 	InterceptDram float64 `csv:"intercept_dram"`
 }
 
+type CoreModelServerCoeff struct {
+	AllWeights struct {
+		BiasWeight           float64 `json:"Bias_Weight"`
+		CategoricalVariables struct {
+			CPUArchitecture struct {
+				AlderLake struct {
```

Comment:
Instead of having the Architecture'a namea encoded in structures, it might be better to have a map where the key is the architecture name and the value is the weight...

Architecture's names can be constants to be used to fill the map.

Reply:
Thank you for the reply! I was thinking that the structs might be better in this case mainly because all of these fields will all be populated and used. The struct is to store a fully populated json request body. I do plan to pull out some of the nested structs and make them into their own types for readability and ease of use. What are your thoughts?

Reply:
Right, but the name of the architectures must be in the variables and not in the name of the fields.
Having the value of variables as field names of the structure is not a good practice, the structure must be generic.

So, we need to have something like:
```golang
type CoreModelServerCoeff struct {
	AllWeights struct {
		BiasWeight           float64 `json:"Bias_Weight"`
		CategoricalVariables struct {
			CPUArchitecture []struct {
				Name   string  `json:"name"`
				Weight float64 `json:"weight"`
			}
		} `json:"cpu_architecture"`
	} `json:"Categorical_Variables"`
}
```

And then the json file will be something like:
```json
{
	"Categorical_Variables": {
		"Bias_Weight": 0,
		"cpu_architecture": [
		{
			"name": "Broadwell",
			"weight": 9
		},
		{
			"name": "Cascade Lake",
			"weight": 127
		}]
}
```

Please, update that accordingly

Reply:
I have updated the structs and incoming json file as requested!